### PR TITLE
Create task_execution.md

### PR DESCRIPTION
**Pull request by ekiernan**
Added best practice doc for using the task execution section.

### CODE REVIEW

**Review by timaeusx — APPROVED** [review]
Left a comment but looks good otherwise!

**Review by nikellepetrillo — APPROVED** [review]
looks great liz! thank you!

File: website/docs/Best_practices/task_execution.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+If a script is small, you can even write scripts in line. The [Resources](#resources) section below links to examples of workflow code that use in-line scripting. If you’re using in-line scripting, remember to keep your code tidy (check out the WARP WDL formatting suggestions). And remember, commands will only work if installed on the environment in which the WDL is running; what works on your local machine will not necessarily work in a virtual machine built off a docker and vice-versa.
```
Comment (jessicaway) [inline website/docs/Best_practices/task_execution.md:10]: perhaps this is obvious, but that can include things like where the software is (path to the jar for instance) and input/output file paths within the docker

**Review by jessicaway — APPROVED** [review]
A couple minor comments, but looks good!

File: website/docs/Best_practices/task_execution.md (hunk 1 above)
Comment (ekiernan) [reply website/docs/Best_practices/task_execution.md]: Thank you! Added this and modified slightly based on Tim's comments about Alpine not always being used as the option.

**Pull request merged**